Framework/LiveData/inc/MantidLiveData/Kafka/KafkaEventListener.h

```diff
@@ -38,8 +43,7 @@ class KafkaEventStreamDecoder;
  */
 class DLLExport KafkaEventListener : public API::LiveListener {
 public:
-  KafkaEventListener();
-  /// Destructor. Should handle termination of any socket connections.
+  KafkaEventListener() = default;
```

Comment:
Very minor point (choose to ignore), but do we even need this line? As there isn't any other constructor defined, the compiler will make this anyway.